package.json

```diff
@@ -26,7 +26,7 @@
     "@0xsequence/relayer": "^0.35.5",
     "@aws/dynamodb-data-mapper": "^0.7.3",
     "@aws/dynamodb-data-mapper-annotations": "^0.7.3",
-    "@badger-dao/sdk": "^1.0.83",
+    "@badger-dao/sdk": "^1.0.84",
```

Comment:
includes update to rewards service to not throw errors if a chain does not have a rewards logger